IntegrationWithPharmacies/MyDateTimeConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace IntegrationWithPharmacies
+{
+    public class MyDateTimeConverter : Newtonsoft.Json.JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(DateTime);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            var t = long.Parse(reader.Value.ToString());
+            return new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc).AddMilliseconds(t);

```

Comment:
The variable 't' must be deleted and its value passed directly to the 'AddMilliseconds ()' method.